protocol/avdl/stellar1/local.avdl

```diff
@@ -117,6 +117,12 @@ protocol local {
   }
   array<CurrencyLocal> getDisplayCurrenciesLocal(int sessionID);
 
+  void validateAccountIDLocal(int sessionID, AccountID accountID);
+  void validateSecretKeyLocal(int sessionID, SecretKey secretKey);
+  // Validates a name for a new account. Checks if any accounts have the same name.
+  // If there is an error getting account names, returns true.
+  void validateAccountNameLocal(int sessionID, string name);
```

Comment:
shouldn't these return `boolean`?

Reply:
They return an RPC error if the input failed validation. There is no "something went unexpectedly wrong" value.

Reply:
"If there is an error getting account names, returns true."

Reply:
but that comment was wrong